homeworks/src/ilyarogachov/homework3converter/Calculate.java

```diff
@@ -0,0 +1,20 @@
+package ilyarogachov.homework3converter;
+
+import java.util.IllegalFormatException;
+
+/**
+ * Created by Tom on 25.04.2017.
+ */
+public class Calculate {
+    private Validator f = new Validator();
+    double usd;
+    public double hrivnausd(String name, double hrn){
+        if(f.Validate(name,hrn)){
+            usd = hrn/27;
```

Comment:
Why you hardcoded the rate? by requirements  rate should be passed in the constructor of calculator